lib/Runtime/Base/EtwTrace.cpp

```diff
@@ -387,8 +387,7 @@ void EtwTrace::LogLoopBodyLoadEvent(FunctionBody* body, LoopHeader* loopHeader,
         {
             methodInfo.method_id = iJIT_GetNewMethodID();
             size_t len = utf8::EncodeInto(utf8MethodName, methodName, (charcount_t)methodLength);
-            uint loopNumber = body->GetLoopNumber(loopHeader) + 1;
```

Comment:
If this `loopNumber` is the same as the parameter and that data is contained in `loopHeader`, should we remove that parameter and extract that value from `loopHeader`? Otherwise, is there an assertion that should be made about whether the parameter `loopNumber` matches the value in `body->GetLoopNumber(loopHeader)`?